files/scripts/swss.sh

```diff
@@ -28,7 +28,7 @@ function read_dependent_services()
     fi
 
     if [[ -f ${ETC_SONIC_PATH}/${SERVICE}_multi_inst_dependent ]]; then
-        MULTI_INST_DEPENDENT="${MULTI_INST_DEPENDENT} cat ${ETC_SONIC_PATH}/${SERVICE}_multi_inst_dependent"
+        MULTI_INST_DEPENDENT="${MULTI_INST_DEPENDENT} $(cat ${ETC_SONIC_PATH}/${SERVICE}_multi_inst_dependent)"
```

Comment:
I am thinking we should use DEPENDENT, instead of MULTI_INST_DEPENDENT for macsec.

with DEPENDENT - if swss goes down dependent services go down. eg: macsec service here will go down

with MULTI_INST_DEPENDENT - if swss goes down dependent services go down. In addition if dependent service like macsec go down -- we bring down evering like swss/syncd etc. We would not need that for macsec.

Reply:
https://github.com/sonic-net/sonic-utilities/blob/7704f6378724468417212f9f05c7a371311670cf/sonic_package_manager/service_creator/creator.py#L358-L362

https://github.com/sonic-net/sonic-buildimage/blob/56a679dc09081b41f55e3671a914eb9c7e4b0d8f/rules/functions#L184-L185

I think, A service as a MULTI_INST_DEPENDENT or DEPENDENT depends on the location of xxx_.service.j2.
But right now, the macsec.service.j2 is in the folder of `per_namespace`. I think it's make sense. So, I think macsec as MULTI_INST_DEPENDENT is following the current convention.

Reply:
The variable DEPENDENT and MULTI_ASIC_DEPENDENT -- the functionality wise it is not the same. I think we should change that name to may be MULTI_ASIC_PEER or so.

https://github.com/sonic-net/sonic-buildimage/blob/master/files/scripts/swss.sh#L251, So we it is in MULTI_ASIC_DEPENDENT we add that to the list of containers /usr/bin/docker-wait-any check for exit, and if any of them exit will restart swss service.